### ISSUE TITLE
Creating a Che workspace fails on production - https://openshift.io

### ISSUE THREAD

**Issue by ldimaggi** [openshiftio/openshift.io#1705]
Steps to recreate:
* Create new project/app/quickstart
* Observe creation of new Che workspace fail:
![screenshot from 2017-12-16 21-30-49](https://user-images.githubusercontent.com/642621/34076059-76ea5bba-e2a8-11e7-9830-1d8485f8638e.png)

* Observe error in Che pod log:

```
[che_log.txt](https://github.com/openshiftio/openshift.io/files/1565266/che_log.txt)

2017-12-17 02:25:19,479[ool-22-thread-1]  [ERROR] [oncurrentCompositeLineConsumer 91]   - An error occurred while writing line to the line consumer org.eclipse.che.api.core.util.lineconsumer.ConcurrentFileLineConsumer@15f6a2
--
  | java.nio.charset.UnmappableCharacterException: Input length = 1
  | at java.nio.charset.CoderResult.throwException(CoderResult.java:282)
  | at sun.nio.cs.StreamEncoder.implWrite(StreamEncoder.java:285)
  | at sun.nio.cs.StreamEncoder.write(StreamEncoder.java:125)
  | at java.io.OutputStreamWriter.write(OutputStreamWriter.java:207)
  | at java.io.BufferedWriter.flushBuffer(BufferedWriter.java:129)
  | at java.io.BufferedWriter.flush(BufferedWriter.java:253)
  | at org.eclipse.che.api.core.util.lineconsumer.ConcurrentFileLineConsumer.writeLine(ConcurrentFileLineConsumer.java:57)
  | at org.eclipse.che.api.core.util.lineconsumer.ConcurrentCompositeLineConsumer.writeLine(ConcurrentCompositeLineConsumer.java:82)
  | at org.eclipse.che.api.core.util.CompositeLineConsumer.writeLine(CompositeLineConsumer.java:69)
  | at org.eclipse.che.api.agent.server.launcher.AbstractAgentLauncher$1.writeLine(AbstractAgentLauncher.java:81)
  | at org.eclipse.che.plugin.docker.machine.LogMessagePrinter.process(LogMessagePrinter.java:40)
  | at org.eclipse.che.plugin.docker.machine.LogMessagePrinter.process(LogMessagePrinter.java:22)
  | at org.eclipse.che.plugin.openshift.client.kubernetes.KubernetesOutputAdapter.call(KubernetesOutputAdapter.java:61)
  | at org.eclipse.che.plugin.openshift.client.kubernetes.KubernetesOutputAdapter.call(KubernetesOutputAdapter.java:28)
  | at io.fabric8.kubernetes.client.utils.InputStreamPumper.run(InputStreamPumper.java:60)
  | at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  | at java.util.concurrent.FutureTask.run(FutureTask.java:266)
  | at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  | at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  | at java.lang.Thread.run(Thread.java:748)


```

**Comment by ldimaggi** [comment]
[che_log.txt](https://github.com/openshiftio/openshift.io/files/1565268/che_log.txt)

**Comment by l0rd** [comment]
This was a bug that has been introduced friday morning and fixed a few hour later (with version `2.0.78`). @ldimaggi I will close that because I'm quite confident that's not reproducible but don't hesitate to re-open it.

**Comment by ldimaggi** [comment]
Thanks - tried this on Sunday night - looks OK!
